static/js/stream_edit.js

```diff
@@ -576,6 +576,34 @@ $(function () {
         e.stopPropagation();
     });
 
+    $("#subscriptions_table").on("click", ".deactivate", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        var stream_name = stream_data.maybe_get_stream_name(get_stream_id(e.target));
+        var stream_id = get_stream_id(e.target);
```

Comment:
This feels like an overly convoluted path to get the stream name and ID.  And I think we want to use something here that throws an error if the stream Id doesn't exist, since that shouldn't happen.


And we shouldn't call `get_stream_id` twice!

Reply:
Done.